mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  let taxedPrice = price * 120 / 100;
+  return taxedPrice;
+}
 
```

Comment:
Based on the name of this function `calculateSalesTax`, I would expect that it would return the amount of sales tax for the item, not the cost of the item including sales tax. If I wanted the cost of the item with tax, i'd probably name it `calculatePriceWithTax'.

The names of the function should indicate what it does. You can use this as a clue to what you should write.